share/tools/create_manpage_completions.py

```diff
@@ -857,11 +857,19 @@ def parse_and_output_man_pages(paths, output_directory, show_progress):
 def get_paths_from_manpath():
     # Return all the paths to man(1) and man(8) files in the manpath
     import subprocess, os
-    proc = subprocess.Popen(['manpath'], stdout=subprocess.PIPE)
+    # Some systems have manpath, others have `man --path` (like Haiku).
+    # TODO: Deal with systems that have neither (OpenBSD)
+    for prog in [['manpath'], ['man', '--path']]:
+        try:
+            program = prog
```

Comment:
Leftover from when I printed the last program that was attempted.

I didn't want to check if python keeps loop variables set after the loop, so I assigned it to another var.